tools/debian/cockpit-networkmanager.install

```diff
@@ -1 +1,2 @@
 usr/share/cockpit/networkmanager/
+usr/share/metainfo/org.cockpit-project.cockpit-networkmanager.metainfo.xml
```

Comment:
So, no `dpkg-vendor --is-ubuntu` after all? (in debian/rules). I don't have a strong opinion either way, but if you want to do that, I'm happy to assist.

Reply:
I didn't see what it would offer us to exclude Ubuntu, cockpit-networkmanager is still offered as regular package in the repos, so why special case it.

Reply:
Yes, agreed.